docs/examples/basic.md

```diff
@@ -0,0 +1,46 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+Basic Usage
+===========
+
+We'll start by importing the always existing, global `Timings` object.
+
+```{code-cell}
+
+from pytimings.timer import global_timings
+global_timings
+```
+
+To measure the execution time of a given code segment you can
+start a timer with a given section name and stop it afterwards.
+`stop` returns a `TimingsDelta` object with the gathered info.
+
+```{code-cell}
+
+global_timings.start("a_section_name")
+...
+global_timings.stop("a_section_name")
+
+```
+
+To inspect deltas of known sections at any other point
+use the `delta` function.
+
+```{code-cell}
+
+global_timings.delta("a_section_name")
+```
+
```

Comment:
perhaps also mention `global_timings.walltime("a_section_name")` or add some hint that there are more helpful functions in `timer.py`?

Reply:
LInk into the api docs maybe?